stack/build.Dockerfile

```diff
@@ -4,7 +4,8 @@ ARG sources
 ARG packages
 ARG package_args='--no-install-recommends'
 
-RUN echo "$sources" > /etc/apt/sources.list
+RUN echo "$sources" > /etc/apt/sources.lis
```

Comment:
@kulhadia I think there's a typo here: `list` -> `lis`.

I don't think this line needs to change at all though.